decidim-elections/app/assets/javascripts/decidim/elections/vote.js.es6

```diff
@@ -104,6 +104,25 @@ $(() => {
     }, 3000)
   })
 
+  // exit message before confirming
+  const $form = $(".evote__options");
+  if ($form.length > 0) {
+    $(document).on("click", "a.confirm", (event) => {
+      window.confirmed = true;
```

Comment:
Setting the `confirmed` variable here would allow you to exit before the vote is actually cast (during the "encrypting..." step). It would be better to do it after the vote is cast. Also, renaming this variable to `vote_cast` could be a way to improve readability.

Reply:
Nice. I'll change it! 👍